README.md

```diff
@@ -39,10 +39,12 @@ The default layout allows to write in English, French, German and Esperanto easi
 Install
 --------------------------------------------------------------------------------
 
-You’ll need the latest version of [Kalamine][2] to build your own layout:
+You’ll need the latest version of [Kalamine][2] to build your own layout. By far
+the easiest way to install kalamine is to use [pipx][5], install it then run the
+following line :
 
 ```bash
-pip install kalamine
+pipx install kalamine
```

Comment:
`pipx` is not easier than `pip` since it’s not install by default on most distros. I’d suggest to say something like:

> we highly recommend using `pipx`: (link)

> then install kalamine like this:
```bash
pipx ensurepath
pipx install kalamine
```

Reply:
I thought saying "the easiest way" was justified, seeing how horrible Kalamine is to install on Arch, but you’re probably right, it’s gonna be harder for people using a normal, non‑complicated distro that ~ suddenly ~ realised that letting the user fucking implode the whole OS with a routine update was a bad idea, actually (ie, a lot of people)